docs/validator/staking.md

```diff
@@ -24,9 +36,9 @@ At least 50 GB free disk
 
 **IMPORTANT: Make sure you have the latest version of NEAR Shell and Node Version; 10.x**
 
-If this is not the case, follow the steps below to set up your environment; don't worry this won't take long. To stake, make sure that you have
+If this is not the case, follow the setps below to set up your environment; don't worry this won't take long. To stake, make sure that you have
```

Comment:
this was corrected recently but seems like you didn't update your local copy, @jimmy3dita .  no problem, it was the only error in a previously merged PR.  just need to fix it here